cmd/liqoctl/cmd/offload.go

```diff
@@ -98,23 +104,29 @@ func newOffloadNamespaceCommand(ctx context.Context, f *factory.Factory) *cobra.
 	options := offload.Options{Factory: f}
 	cmd := &cobra.Command{
 		Use:     "namespace name",
-		Aliases: []string{"ns"},
-		Short:   "Offload a namespace to remote clusters",
+		Aliases: []string{"ns", "namespaces"},
+		Short:   "Offload namespaces to remote clusters",
 		Long:    liqoctlOffloadNamespaceLongHelp,
 
-		Args:              cobra.ExactArgs(1),
 		ValidArgsFunction: completion.Namespaces(ctx, f, 1),
```

Comment:
To enable suggestions for more than one arg
```suggestion
		ValidArgsFunction: completion.Namespaces(ctx, f, completion.NoLimit),
```